guides/v2.1/config-guide/bootstrap/magento-bootstrap.md

```diff
@@ -14,20 +14,20 @@ functional_areas:
   - Setup
 ---
 
-<h2 id="config-boot-overview">Overview of bootstrapping</h2>
-To run the Magento application, the following actions are implemented in <a href="{{ site.mage2000url }}index.php" target="_blank">index.php</a>:
+## Overview of bootstrapping {#config-boot-overview}
+To run the Magento application, the following actions are implemented in [index.php]({{ site.mage2000url }}index.php){:target="&#95;blank"}:
 
-*	Include <a href="{{ site.mage2000url }}app/bootstrap.php" target="_blank">app/bootstrap.php</a> which performs essential initialization routines, such as error handling, initializing the autoloader, setting profiling options, setting the default timezone, and so on.
-*	Create an instance of <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/App/Bootstrap.php" target="_blank">\Magento\Framework\App\Bootstrap</a>. <!-- It requires initialization parameters to be specified in constructor.  Normally, the $_SERVER super-global variable is supposed to be passed there. -->
-*	Create a Magento application instance (<a href="{{ site.mage2000url }}lib/internal/Magento/Framework/AppInterface.php" target="_blank">\Magento\Framework\AppInterface</a>).
+*	Include [app/bootstrap.php]({{ site.mage2000url }}app/bootstrap.php){:target="&#95;blank"} which performs essential initialization routines, such as error handling, initializing the autoloader, setting profiling options, setting the default timezone, and so on.
+*	Create an instance of [\\Magento\\Framework\\App\\Bootstrap]({{ site.mage2000url }}lib/internal/Magento/Framework/App/Bootstrap.php){:target="&#95;blank"}. <!-- It requires initialization parameters to be specified in constructor.  Normally, the $_SERVER super-global variable is supposed to be passed there. -->
```

Comment:
Remove the double slashes. Also please ensure they don't exist anywhere else in your changes.